python/ModelTools.py

```diff
@@ -494,7 +497,7 @@ def doExpectedEvents(self):
 		    argu = self.DC.rateParams["%sAND%s"%(b,p)][rk][0][0]
 		    if self.out.arg(argu): factors.append(argu)
 		    else: raise RuntimeError, "No rate parameter found %s, are you sure you defined it correctly in the datacard?"%(argu)
-
+                selfNormRate = 1.0
```

Comment:
For RoolRealSumPdf channels we don't put the nominal yield in the ProcessNormalisation